front/app/api/navbar/util.ts

```diff
@@ -33,8 +33,8 @@ export function getNavbarItemSlug({
   }
 
   // This is impossible, but I can't seem to make typescript understand
-  // that. So just returning null here
-  return null;
+  // that. So just returning the homepage here
+  return '/';
```

Comment:
Not sure if this is the cleanest solution, but it was the easiest I could think of 😅 Perhaps some kind of filter_map in SiteMap/index.tsx would have been a better approach?